README.md

```diff
@@ -9,6 +9,10 @@ Prerequisites: autotools, pkg-config, libtool, gcc, clang or msys2/mingw32
     ./bootstrap.sh
     ./configure
     make && make install
+### Note for windows compilation
```

Comment:
I am all for dumping autotools at this point.  I am over it.  It is really powerful and really painful where something like cmake provides everything this project needs in a simpler package.  There should be a CMake file floating around here somewhere that builds for likes and nothing else.  If you can get that work it would be awesome.